matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/EventRelationsAggregationTask.kt

```diff
@@ -42,6 +44,14 @@ internal interface EventRelationsAggregationTask : Task<EventRelationsAggregatio
     )
 }
 
+enum class VerificationState {
+    REQUEST,
```

Comment:
`REQUESTED` for coherency with other values?

Reply:
will fix on next PR